enums/routes.ts

```diff
@@ -0,0 +1,7 @@
+export enum Routes {
+    HOME = '/',
+    DASHBOARD = '/dashboard',
+    SIGNIN = '/signin',
+    SIGNOUT = '/signout',
+    LOGIN = '/login',
+  }
```

Comment:
Are you going to have a direct route to the page that shows/edits a NR? That would be nice.

Reply:
I planning to add a direct route to the examine names page